relay-config/src/config.rs

```diff
@@ -1749,6 +1754,13 @@ impl Config {
         Duration::from_secs(self.values.limits.shutdown_timeout)
     }
 
+    /// Set server keep-alive setting.
+    ///
+    /// By default keep alive is set to a 5 seconds.
+    pub fn keepalive_timeout(&self) -> usize {
```

Comment:
```suggestion
    /// Returns the server keep-alive timeout in seconds.
    ///
    /// By default keep alive is set to a 5 seconds.
    pub fn keepalive_timeout(&self) -> usize {
```

Note that we expose `Duration` in other methods and then map it back to seconds at the usage site. That's a redundant roundtrip, but also reduces the chance of accidentally misinterpreting the unit of the number at a different place.